src/components/scenes/FioAddressDetailsScene.js

```diff
@@ -60,23 +61,40 @@ class FioAddressDetails extends React.Component<Props, LocalState> {
 
   _onPressAccountSettings = (): void => {
     const { navigation, route } = this.props
-    const { fioAddressName } = route.params
+    const { fioAddressName, bundles } = route.params
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name